projects/optic/src/commands/diff/diff.ts

```diff
@@ -48,15 +48,15 @@ type DiffActionOptions = {
   generated?: boolean;
 };
 
-const description = `run a diff between two API specs`;
+const description = 'Run a diff between two API specs';
 
 const usage = () => `
   optic diff <file_path> --base <base>
-  optic diff <file_to_compare_against> <file_path>
-  optic diff <file_to_compare_against> <file_path> --check`;
+  optic diff [file_to_compare_against] <file_path>
+  optic diff [file_to_compare_against] <file_path> --check`;
```

Comment:
this is informational, but im following commander's syntax here and differentiating optional vs required arguments.